### ISSUE TITLE
Kudos not displayed on single post

### ISSUE THREAD

**Issue by jfmartin67** [ericgregorich/micro-blog-cards-theme#17]
Option is check on plugin settings page. 
Tinylytics plugin is installed and configured. 
![Safari-Personal (Default) — The work to switch my blog’s visual theme is …@2024-01-28@21 09 58@2x](https://github.com/ericgregorich/micro-blog-cards-theme/assets/211377/27ed5caf-7f3e-4135-9edb-70913127296c)

**Comment by ericgregorich** [comment]
Are you using a plugin to add Tinylytics or adding your embed script directly into your footer?

My blog is running 1.6 and I did not have Tinylytics setup. I added my embed code to Design > Edit Footer.

`<script src="https://tinylytics.app/embed/{YourEmbedCode}.js?kudos" defer></script>`

After saving, I went to the Cards Theme Plugin and enabled the " Enable Tinylytics Kudos?" checkbox and saved.

A few seconds later, Kudos started showing on my posts.

![image](https://github.com/ericgregorich/micro-blog-cards-theme/assets/26981280/7a530cc9-e704-4edf-af86-f39c7207c2c1)


Let me know if your using a Plugin instead. Perhaps something else is conflicting with it on you page.

**Comment by ericgregorich** [comment]
I ran one of your posts through an [HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fblog.numericcitizen.me%2F2024%2F01%2F30%2Fthis-is-such.html) and found some errors that appear to be coming from your custom Footer HTML. 

I see your Tinylytics embed code in there, but it may not be rendering correctly since some of your tags seem to be out of order.

**Comment by jfmartin67** [comment]
Actually, I do have the plugin AND the embed code in the footer... removed the plugin, waiting for site to rebuild... let you know.

**Comment by jfmartin67** [comment]
Well, I removed all the but Tinylytics embed line in the footer, the plugin was uninstalled and enabled the show kudos.. the HTML validator still returns some errors...

**Comment by ericgregorich** [comment]
If you remove everything from your footer except the Tinylytics script, does it render? Does the validation errors go way?

I still feel like it may be related some of the HTML in your footer causing some issue. Removing everything except the script may prove or disprove that theory.

**Comment by jfmartin67** [comment]
Nope. Nokus. The following is the current error state. I get the feeling that this is something within the theme itself... 
![Safari-Personal (Default) — Showing results for httpsblog numericcitizen me20240130this-is-such html - Nu Html Checker@2024-01-31@06 18 06@2x](https://github.com/ericgregorich/micro-blog-cards-theme/assets/211377/46fd9eb6-8636-4e5f-9edd-9fb6c6a30259)

**Comment by ericgregorich** [comment]
Please see if there is any difference in v1.7. 

I'm not seeing any validation errors on my blog except on posts that don't have conversation. In that case, the conversation. In returns an error. I need to check with Manton about that.

**Comment by jfmartin67** [comment]
1.7, nor 1.8 fixes the issue.

**Comment by jfmartin67** [comment]
Kudos are working because I decided to install the Tinylytics Plugin. But I'm working at showing the webring and hits.
